modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/rule/HashAggregateConverterRule.java

```diff
@@ -84,8 +88,11 @@ private static class MapReduceHashAggregateConverterRule extends AbstractIgniteC
         }
 
         /** {@inheritDoc} */
-        @Override protected PhysicalNode convert(RelOptPlanner planner, RelMetadataQuery mq,
-            LogicalAggregate agg) {
+        @Override protected @Nullable PhysicalNode convert(
```

Comment:
Trivial codestyle. Might be a sigle line. Up to you.

Reply:
Exceeds line limit.